android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java

```diff
@@ -106,6 +109,12 @@ public void onCreate() {
         }
         refWatcher = LeakCanary.install(this);
 
+        //Initialize Cache
+        File httpCacheDirectory = new File(getCacheDir(), "picasso-cache");
+        Cache cache = new Cache(httpCacheDirectory, 15 * 1024 * 1024);
+
```

Comment:
This will just feed the max cache size to the LruCache class, and it will handle space management by itself. It won't create the 10 MB file on opening of the app